recipes/slacker/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.9.16" %}
+
+package:
+    name: tinydb
+    version: {{ version }}
+
+source:
+    url: https://github.com/os/slacker/archive/v{{ version }}.tar.gz
+    fn: slacker-{{ version }}.tar.gz
+    sha256: 453d554adcc33df92db926fc4f5ad2a5a10364cdae4503ec1ca67bf4a8684592
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - setuptools
+        - python
+    run:
+        - python
+        - requests
+
+test:
+    imports:
+        - slacker
+    requires:
+        - pytest
+        - mock
+    commands:
+        - py.test {{ environ.SRC_DIR }}/tests -vrsx
+
+about:
+    home: https://github.com/os/slacker
+    license: Apache
```

Comment:
@ericdill please add the version here: https://github.com/os/slacker/blob/master/LICENSE#L2